### PR TITLE
Update 168 paid master

### PR DESCRIPTION
**Pull request by oskarth**
Still not as specced out as I would like, but I think we have enough to get started, even with risks.

How about an initial kick off call on Monday where we can sync tech MVP and UX(R) track?

### CODE REVIEW

File: ideas/168-paid-master-nodes/README.md [hunk 1]
```diff
@@ -16,14 +38,22 @@ users.
...
+- Not enough Go and Clojure resources available
+
+Counteract:
+- Create clear issues as bounties
+- Oskar start initial spiking out, be OK with slow progress
+- Proactively reach out to individual devs to try to free up resources
+
 ## Product Overview
 
 This idea takes of where https://github.com/status-im/ideas/blob/master/ideas/1-offline-inboxing.md ended. It provides an SNT payment layer for usage of master nodes.
```
Comment (andytudhope) [inline ideas/168-paid-master-nodes/README.md]: `off`

File: ideas/168-paid-master-nodes/README.md [hunk 2]
```diff
@@ -85,7 +115,9 @@ whitelisting this user's enode for some period of time.
 
 How re-newal will happen is not specified yet.
```
Comment (andytudhope) [inline ideas/168-paid-master-nodes/README.md]: `renewal`

File: ideas/168-paid-master-nodes/README.md [hunk 3]
```diff
@@ -85,7 +115,9 @@ whitelisting this user's enode for some period of time.
...
 
-> Comment from review: May be out of scope for this idea but perhaps we'll want to experiment with different payment models for time - e.g. Alice pays until she runs out of SNT OR N offline messages OR set period of time.
+> Comment from review: May be out of scope for this idea but perhaps we'll want
+> to experiment with different payment models for time - e.g. Alice pays until
+> she runs out of SNT OR N offline messages OR set period of time.
 
 Solving the initial user stories don't require any proof of message delivery or
 quality of service. Reputation of specific mail servers thus happens completely
```
Comment (andytudhope) [inline ideas/168-paid-master-nodes/README.md]: No idea what `Reputation of specific mail servers thus happens completely` means?

File: ideas/168-paid-master-nodes/README.md (hunk 1 above)
Comment (oskarth) [reply ideas/168-paid-master-nodes/README.md]: `oeuf`

File: ideas/168-paid-master-nodes/README.md (hunk 3 above)
Comment (oskarth) [reply ideas/168-paid-master-nodes/README.md]: Does this make it clearer? https://github.com/status-im/ideas/pull/216/commits/b02330365e613860396411e0841d31142242e507

**Pull request merged**